### ISSUE TITLE
BinjaDock?

### ISSUE THREAD

**Issue by agnosticlines** [Vector35/binaryninja-plugins#17]
Hey there,

As far as I'm aware Binja doesn't have a native plugin API that can add GUI elements right?

There's a plugin that does this here: https://github.com/NOPDev/BinjaDock (Note: not mine)

I'd be happy to package it up and submit a pull request, just wanted to know if I had permission to do that first or if this plugin is now not needed due to native plugin GUI support I just haven't found/used yet?

Love the project,

Thanks. :-)

**Comment by lucasduffey** [comment]
Duplicate of https://github.com/Vector35/binaryninja-api/issues/305

**Comment by psifertex** [comment]
No, he's specifically talking about adding the third party plugin to the plugin repo, not just the functionality itself.

I'm a bit torn since we don't want to encourage too much use of the other plugin as the requirements are much more complicated and would likely never be properly installable through the one-click UI installer we plan for the plugin repo, but at the same time, it /is/ a useful plugin.

I think it's fine, go ahead and submit. I'd recommend first issuing a PR to NOPDev to see if you can get the original plugin repo updated with the new metadata first to avoid unnecessary forks unless they're unwilling to accept the PR.

It'd be great if you could update the install instructions for it too.

**Comment by psifertex** [comment]
@agnosticlines Just following up to see if you've had a chance to issue the pull request? Even with the extra setup, be happy to have the plugin included here, we definitely want to get as many as possible added gearing up to the release of the API that will use it for plugin management.
